docusaurus.config.js

```diff
@@ -210,6 +210,7 @@ const config = {
             prism: {
                 theme: lightCodeTheme,
                 darkTheme: darkCodeTheme,
+                additionalLanguages: ['rust', 'bash', 'typescript', 'csharp', 'lua', 'protobuf', 'powershell', 'toml', 'yaml'],
```

Comment:
Please also add the `wasm`, `javascript`, and `go`, thanks.

Reply:
Okay sure :)